typings/built-in-glamorous-components.d.ts

```diff
@@ -0,0 +1,152 @@
+import { ExtraGlamorousProps } from './glamorous-component';
+import { CSSProperties } from './css-properties';
+
+/*
+ * FIXME:
+ * Since TypeScript doesn't have
+ * HTMLDetailsElement, HTMLDialogElement,
+ * HTMLKeygenElement, HTMLMenuItemElement
+ * Those components currently has wrong type.
+ * After TypeScript add those types, plz fix this.
```

Comment:
Would you mind adding a link to the tracking issue for this to the comment.

https://github.com/Microsoft/TypeScript/issues/17828